### PR TITLE
Move compiler installation to 1 language per file

### PR DESCRIPTION
**Pull request by RabsRincon**
Some times it's a bit of a pain to have to install every compiler if you want only one (You have to get into the script and comment out everything not needed)
This should make a better way to have some granularity control over which compilers are to be installed.

I need PR review because my bash-fu is not something I'm secure about. Please find every bug the change might have introduced.

### CODE REVIEW

**Comment by mattgodbolt** [comment]
Hahaha! Will take a look....I'm not against using python or even Make for this stuff for what it's worth...

File: update_compilers/install_assembly_compilers.sh [hunk 1]
```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+
+SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+. ${SCRIPT_DIR}/common.inc
+
+ARG1="$1"
+
+install_nightly() {
```
Comment (mattgodbolt) [inline update_compilers/install_assembly_compilers.sh]: seems like this is a candidate to put in `common.inc` ?

File: update_compilers/install_assembly_compilers.sh [hunk 2]
```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+
+ARG1="$1"
```
Comment (mattgodbolt) [inline update_compilers/install_assembly_compilers.sh]: is this still needed? Seems it's in `common.inc` ?

Comment (RabsRincon) [reply update_compilers/install_assembly_compilers.sh]: The idea is that you can also run each file separatedly, and I did not find a better way to ensure I also can call `install_*_compilers.sh nightly` properly.
Would `. ${SOURCE_DIR}/common.inc "$@"` work?

**Pull request merged**